samples/subsys/lorawan/class_a/overlay-multiregion.conf

```diff
@@ -0,0 +1,2 @@
+CONFIG_LORAMAC_REGION_EU868=y
+CONFIG_LORAMAC_REGION_US915=y
```

Comment:
No need for a new overlay, you can create a new testcase next to existing one (check `sample.yaml`) so that the multi-region config gets picked up by the CI and we actually build this code as part of it.

Reply:
like this?

Reply:
Yup, that's it. Thanks for the change.